### ISSUE TITLE
loosing AKT

### ISSUE THREAD

**Issue by alyasboytigas** [HandyOSS/HandyHost#81]
Hello,

I just updated my handyhost v0.5.5 and migrated akash v0.1.6.3 per previous instruction and finally it's working.

I've been checking my AKT and it's slowly going down. Today when I update I lost almost 70 AKT.

![Screenshot (333)](https://user-images.githubusercontent.com/57547739/165545775-772b50ad-e210-4bfc-a97d-23b1707a3bec.png)
![Screenshot (372)](https://user-images.githubusercontent.com/57547739/165545789-732e9d84-7ea4-40ee-8627-ec815016d2c0.png)
![Screenshot (385)](https://user-images.githubusercontent.com/57547739/165545810-6dd1ae35-4f33-4346-b331-49fd8d42d5a0.png)

Thanks.

**Comment by alyasboytigas** [comment]
![156202032-8cf266b3-721b-49d3-8501-17e114e1dc09 (1)](https://user-images.githubusercontent.com/57547739/165547906-340c5416-b390-4cbe-a27e-eb623c6bea1b.png)

**Comment by alexsmith540** [comment]
If possible can you email me your public wallet address to ***@***.***?
That way I can dig into the transactions and my rpc endpoint that does the dashboard aggregate counts and see what’s going on. 
The scenario I see would be that you actually could have a bunch of active leases but the UI isn’t updating the aggregates counts.
In any case your funds are safe on chain somewhere whether in deposits or escrow, the UI just might not be updating right after that big akash update.
Thanks again for the heads up.

**Comment by alyasboytigas** [comment]
thanks alex i emailed you my address at your yahoo email account.

**Comment by alexsmith540** [comment]
Cool, received. 
One thing I have noticed digging in on mint scan is that the deposit amount may have risen from 5akt to 50akt. Not confirmed yet but I think this is what’s going on. In the handyhost UI we display the locked deposit akt amount by doing (active + open orders) * 5akt since it’s faster than querying each transaction for the deposit amount so if the deposit changed from 5akt to 50akt that’s why.
Regardless I’ll dig in to confirm and likely push a UI change later today if it’s the case the deposit amount changed.

**Comment by avolon42x** [comment]
Observed something similar - had two open bids and suddenly my balanced tanked 100 akt.

If you're planning an update any chance of adding the "dynamic gas adjustments": https://discord.com/channels/747885925232672829/771909807359262751/968543966972751892

**Comment by alexsmith540** [comment]
Confirmed they moved the deposit amount up to 50AKT in v0.16.3
I'm not sure if that was intentional or not since the docs all still say 5AKT for deposits but regardless i'll update the UI to reflect the 50AKT deposit price.
From ```bin/akash provider run --help``` in v0.16.3
```--bid-deposit string                          Bid deposit amount (default "50000000uakt")```

**Comment by alexsmith540** [comment]
> Observed something similar - had two open bids and suddenly my balanced tanked 100 akt.
> 
> If you're planning an update any chance of adding the "dynamic gas adjustments": https://discord.com/channels/747885925232672829/771909807359262751/968543966972751892

Yeah i'll look into it, should be easy enough to add. Will make all those out of gas tx errors more minimal i'd hope.

**Comment by alexsmith540** [comment]
Yeah this PR makes me feel like it was a mistake::

https://github.com/ovrclk/akash/commit/17a45bfbab21c7e3d36cffdc186ce906a1acc645

Like they’re matching the 5akt deposits on tests but now on mainnet we have 50akt deposits I don’t get it lol.
I really hope it’s a mistake, putting down a 50akt deposit for an 0.1cpu (or even a few cpu) contract seems like overkill.

**Comment by avolon42x** [comment]
I added that now to the providerRun.sh but maybe you can change that as well?
https://discord.com/channels/747885925232672829/771909807359262751/968935314280300554

**Comment by alexsmith540** [comment]
FYI I just patched the runProvider script to use 5AKT for deposits, should be all set after you run the v0.5.6 installer and restart your provider.

**Comment by alexsmith540** [comment]
I'll look into the dynamic gas adjustments next, wanted to get this patch out the door asap.

**Comment by alyasboytigas** [comment]
looks like it's back to normal...

Just wondering tho, I started somewhere 94 or 95 AKT now I have 85AKT is that normal?

![Screenshot (389)](https://user-images.githubusercontent.com/57547739/166170817-2ec42057-34d9-435d-be5c-7c4ee9d6e3b5.png)

**Comment by freakinfofa** [comment]
see issue #66 it might help you if you have not changed the default settings.

**Comment by alexsmith540** [comment]
You’re good to go, only HandyHost version bumped in this case. Once those bids with the 50akt deposit expire you’ll get your 50akt deposit refunded. And any new bids from here out are 5akt deposit.
